website/docs/components/form/file-input/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,31 @@
+## Usage
+
+### When to use
+
+- Use a file input to let users quickly select files from their computer.
```

Comment:
Should this be capitalized? Is it referencing the component, or `type=file` (I know, fine line here)

Additionally, probably best to use "device" here. Nit-picky, but probably relevant for users on a tablet, etc. (forward thinking here)

```suggestion
- Use a File Input to let users quickly select files from their device.
```

Reply:
@jorytindall should it be "the File Input" if we're referencing the component? or "a File Input component"?

Reply:
The age old question lol. I'm not sure if one is more correct over the other, but "the File Input component" reads a better to me 🤔

Reply:
Probably "the File Input" component I think since it's talking about a specific HDS component